enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -306,6 +312,7 @@ class Meta:
     group_uuid = models.UUIDField(null=True, blank=True, db_index=True)
     catalog_uuid = models.UUIDField(null=True, blank=True, db_index=True)
     access_method = AccessMethods.DIRECT
+    history = HistoricalRecords()
```

Comment:
You're gonna want to remove this one and the ones for the other policy models below - this is a vestige of our prototype code and doesn't belong in this repo.
This PR: https://github.com/openedx/enterprise-subsidy/pull/2 will drop those models.

For your PR, I'd:
1. undo your local migration `./manage.py migrate subsidy 0001`
2. remove the current subsidy 0002... migration file
3. remove these `history` fields you've added for the "Policy" models
4. regenerate the migration file `./manage.py makemigrations ... && ./manage.py migrate subsidy`